cli/src/tasks.rs

```diff
@@ -38,9 +38,11 @@ pub enum TaskAction {
         time_precision: Duration,
         #[arg(long)]
         hpke_config_id: Uuid,
-        #[arg(long, required_if_eq("vdaf", "histogram"), value_delimiter = ',')]
-        buckets: Option<Vec<u64>>,
-        #[arg(long, required_if_eq_any([("vdaf", "count_vec"), ("vdaf", "sum_vec")]))]
+        #[arg(long, value_delimiter = ',')]
+        categorical_buckets: Option<Vec<String>>,
+        #[arg(long, value_delimiter = ',')]
+        continuous_buckets: Option<Vec<u64>>,
+        #[arg(long, required_if_eq_any([("vdaf", "count_vec"), ("vdaf", "sum_vec"), ("vdaf", "histogram")]))]
         length: Option<u64>,
```

Comment:
I think requiring this for histograms at the clap level is too strict, as it would make the above buckets flags moot.